src/main/java/seedu/address/model/Model.java

```diff
@@ -14,8 +14,11 @@
     /** {@code Predicate} that always evaluate to true */
     Predicate<Person> PREDICATE_SHOW_ALL_PERSONS = unused -> true;
 
+    /** {@code Predicate} that always evaluate to false */
+    Predicate<Person> PREDICATE_SHOW_EMPTY_LIST = unused -> false;
+
     /**
-     * Replaces user prefs data with the data in {@code userPrefs}.
+     * * Replaces user prefs data with the data in {@code userPrefs}.
```

Comment:
Just a veryvery minor point on deleting the extra `*` in the javadoc comment! :sweat_smile:

Reply:
Oh my I totally have no idea why this happened. Thanks for pointing it out!